### PR TITLE
Remove tests related to web app routing as it is no longer supported.

### PR DESCRIPTION
**Pull request by chtyim**
Critical fix for broken test by https://github.com/caskdata/cdap/pull/7418

### CODE REVIEW

**Comment by chtyim** [comment]
The affected test `NettyRouterHttpsTest` and `NettyRouterHttpTest` both passed locally.

**Comment by anwar6953** [comment]
LGTM

File: cdap-gateway/src/test/java/co/cask/cdap/gateway/router/NettyRouterTestBase.java [hunk 1]
```diff
@@ -113,53 +107,10 @@ public String get() {
...
   };
 
-  private final Supplier<String> webappServiceSupplier = new Supplier<String>() {
-    @Override
-    public String get() {
-      try {
-        return WEB_APP_SERVICE_PREFIX + Networks.normalizeWebappDiscoveryName(HOSTNAME + ":" +
-                                                                                lookupService(WEBAPP_SERVICE));
-      } catch (UnsupportedEncodingException e) {
-        LOG.error("Got exception: ", e);
```
Comment (anwar6953) [inline cdap-gateway/src/test/java/co/cask/cdap/gateway/router/NettyRouterTestBase.java:123]: ah, the good ol' log and rethrow

Comment (chtyim) [reply cdap-gateway/src/test/java/co/cask/cdap/gateway/router/NettyRouterTestBase.java:123]: these code hide in the deepest corner.

**Pull request merged**